tests/semanticdb/metac.expect

```diff
@@ -49,44 +49,47 @@ Schema => SemanticDB v4
 Uri => Advanced.scala
 Text => empty
 Language => Scala
-Symbols => 34 entries
-Occurrences => 90 entries
+Symbols => 37 entries
+Occurrences => 96 entries
 
 Symbols:
-advanced/C# => class C
-advanced/C#[T] => typeparam T
-advanced/C#`<init>`(). => primary ctor <init>
-advanced/C#t(). => method t
-advanced/Structural# => class Structural
-advanced/Structural#`<init>`(). => primary ctor <init>
-advanced/Structural#s1(). => method s1
-advanced/Structural#s2(). => method s2
-advanced/Structural#s3(). => method s3
-advanced/Test. => final object Test
-advanced/Test.e. => val method e
-advanced/Test.e1. => val method e1
-advanced/Test.e1x. => val method e1x
-advanced/Test.s. => val method s
-advanced/Test.s1. => val method s1
-advanced/Test.s1x. => val method s1x
-advanced/Test.s2. => val method s2
-advanced/Test.s2x. => val method s2x
-advanced/Test.s3. => val method s3
-advanced/Test.s3x. => val method s3x
-advanced/Wildcards# => class Wildcards
-advanced/Wildcards#`<init>`(). => primary ctor <init>
-advanced/Wildcards#e1(). => method e1
-local0 => abstract val method x
-local1 => abstract val method x
-local2 => final class $anon
-local3 => val method x
-local4 => abstract method m
-local5 => param x
-local6 => final class $anon
-local7 => method m
-local8 => param x
-local9 => val local e3
-local10 => val local e3x
+advanced/C# => class C [typeparam T ] extends Object { self: C[T] => +3 decls }
+advanced/C#[T] => typeparam T 
+advanced/C#`<init>`(). => primary ctor <init> [unknown T: <?>]: C[T]
+advanced/C#t(). => method t => T
+advanced/Structural# => class Structural extends Object { self: Structural => +5 decls }
+advanced/Structural#`<init>`(). => primary ctor <init> (): Structural
+advanced/Structural#s1(). => method s1 => Object { abstract val method x Int }
+advanced/Structural#s2(). => method s2 => Object { abstract val method x Int }
+advanced/Structural#s3(). => method s3 => Object { abstract method m (param x: Int): Int }
+advanced/Structural#s4(). => method s4 (param a: Int): Object { abstract val method x Int }
+advanced/Structural#s4().(a) => param a: Int
+advanced/Test. => final object Test extends Object { self: <?>.Test.type => +11 decls }
+advanced/Test.e. => val method e Wildcards
+advanced/Test.e1. => val method e1 List[<?>]
```

Comment:
wildcard type arguments (`TypeBounds` in argument position) should be converted to `ExistentialType`, it seems that they are just unhandled presently

Reply:
done https://github.com/lampepfl/dotty/pull/12885/commits/f509613cfd5ece7bd31789c57c907d2388040763